llarp/lokinet_shared.cpp

```diff
@@ -525,6 +536,12 @@ extern "C"
     return new lokinet_context{};
   }
 
+  struct lokinet_context* EXPORT
+  lokinet_context_from_config(struct lokinet_config* cfg)
+  {
+    return new lokinet_context{cfg ? cfg->impl : nullptr};
```

Comment:
Since the only point of this function is to pass a config, passing NULL here seems like an error, not something that should be silently swallowed into a default embedded config.